Dockerfile

```diff
@@ -166,4 +195,4 @@ RUN mkdir /var/run/sshd
 RUN echo 'root:root' | chpasswd
 RUN sed -ri 's/^PermitRootLogin\s+.*/PermitRootLogin yes/' /etc/ssh/sshd_config
 RUN sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config
-EXPOSE 22
+EXPOSE 22
```

Comment:
A small format issue, add end of line at last line. Other things LGTM

Reply:
Done, thx